lib/src/argument_list_visitor.dart

```diff
@@ -147,14 +147,8 @@ class ArgumentListVisitor {
 
     if (functionsStart == null) {
       // No functions, so there is just a single argument list.
-      return new ArgumentListVisitor._(
-          visitor,
-          leftParenthesis,
-          rightParenthesis,
-          arguments,
-          new ArgumentSublist(arguments, arguments),
-          null,
-          null);
+      return ArgumentListVisitor._(visitor, leftParenthesis, rightParenthesis,
+          arguments, ArgumentSublist(arguments, arguments), null, null);
```

Comment:
Unlike the [below `ArgumentListVisitor._` instantiation (line 160)](https://github.com/dart-lang/dart_style/pull/699/files#diff-0aa3c812b9d05341dfe503408b560ed6R160) it's on 2 lines instead of one argument per line. It's a little strange.

Reply:
Yup. If an argument list fits on two lines, dartfmt packs it. More than two, and it goes to one argument per line.

This invocation, without the `new` is now just short enough to fit everything in two lines.